pages/1.12/installing/production/upgrading/index.md

```diff
@@ -30,10 +30,94 @@ If upgrading is performed on a supported OS with all prerequisites fulfilled, th
 - An upgraded DC/OS Marathon leader cannot connect to the leading Mesos master until it has also been upgraded. The DC/OS UI cannot be trusted until all masters are upgraded. There are multiple Marathon scheduler instances and multiple Mesos masters, each being upgraded, and the Marathon leader may not be the Mesos leader.
 - Task history in the Mesos UI will not persist through the upgrade.
 
-## Supported upgrade paths
-- From the latest GA version of previous to the latest GA version of current. For example, if 1.11.8 is the latest GA version of previous and 1.12.4 is the latest GA version of current, this upgrade would be supported.
+## Supported upgrade paths matrix
+The following matrix table lists the supported upgrade paths for DC/OS 1.11.
+
+
+|**Display Icon** | **Service** |
```

Comment:
@gauripowale I think there is a dangling "<" and some additional <tr>s that are dangling in the table.  Looks like this on my end - 

https://imgur.com/a/WseO6mI

Just need that cleaned up and we're good to go!

Reply:
The table is not formatted well.
<img width="1436" alt="screen shot 2018-10-24 at 6 08 15 pm" src="https://user-images.githubusercontent.com/36425832/47469839-d6f27100-d7b7-11e8-936c-983a175bc608.png">